pkgs/development/libraries/gtk/3.x.nix

```diff
@@ -49,6 +51,8 @@
 , testers
 }:
 
+assert withGtkDoc -> withIntrospection;
```

Comment:
Perhaps just use a single `withIntrospection` flag? The `withGtkDoc` was only introduced for cross-compilation purposes in the first place. I would like to keep the cross-compilation related changes as minimal as possible.

Reply:
It was actually [introduced](https://github.com/NixOS/nixpkgs/pull/72269) to avoid building the documentation on Darwin, and then later modified to also be disabled when cross compiling.  So I'm happy to try getting rid of it, but that problem might not have been resolved.  We'll have to test.